### ISSUE TITLE
Crash when selecting 

### ISSUE THREAD

**Issue by venam** [cwensley/pablodraw#39]
I'm running pablodraw release 3.3.0-alpha on ArchLinux with GTK version 3.24.28.

```
pkg-config --modversion gtk+-3.0
3.24.28
```

The application crashes when selecting an item from the list of works on the left.  
I get the following error:

```
Exception in Gtk# callback delegate
  Note: Applications can use GLib.ExceptionManager.UnhandledException to handle the exception.
System.NullReferenceException: Object reference not set to an instance of an object.
   at Gtk.Widget.DrawnSignalCallback(IntPtr inst, IntPtr arg0, IntPtr gch)
   at GLib.ExceptionManager.RaiseUnhandledException(Exception e, Boolean is_terminal)
   at Gtk.Widget.DrawnSignalCallback(IntPtr inst, IntPtr arg0, IntPtr gch)
   at Eto.GtkSharp.Forms.ApplicationHandler.Run()
   at Eto.Forms.Application.Run()
   at PabloDraw.Program.Run()
   at PabloDraw.Startup.Main()
```

**Comment by cwensley** [comment]
Hey @venam, thanks for reporting the issue.  

This is the same as https://github.com/picoe/Eto/issues/1914, which means it'll need an update to get it working again.  I'll try to get another alpha out!

**Comment by ericolito** [comment]
> Hey @venam, thanks for reporting the issue.
> 
> This is the same as [picoe/Eto#1914](https://github.com/picoe/Eto/issues/1914), which means it'll need an update to get it working again. I'll try to get another alpha out!

Any news on this? :)

**Comment by corcoran** [comment]
Can confirm, also happening here!

**Comment by cwensley** [comment]
This should be fixed in [3.3.0-beta1](https://github.com/cwensley/pablodraw/releases/tag/3.3.0-beta1), give it a try!
